src/inverted_index/inverted_index.c

```diff
@@ -21,13 +21,17 @@
 #include "rmutil/rm_assert.h"
 #include "geo_index.h"
 
-uint64_t TotalIIBlocks = 0;
+uint64_t TotalBlocks = 0;
+
+size_t TotalIIBlocks() {
```

Comment:
Even though it's short term, can you document that this is a temporary function?

Reply:
Done in https://github.com/RediSearch/RediSearch/pull/6850/commits/2d16d6fe4d9ae7389c1812bb038da370c1a8620e